desktop/src/main/java/bisq/desktop/main/dao/governance/ProposalDisplay.java

```diff
@@ -477,6 +486,12 @@ public void applyProposalPayload(Proposal proposal) {
             linkHyperlinkWithIcon.setOnAction(e -> GUIUtil.openWebPage(proposal.getLink()));
         }
 
+        if (txHyperlinkWithIcon != null) {
+            txHyperlinkWithIcon.setText(proposal.getTxId());
+            txHyperlinkWithIcon.setOnAction(e ->
+                    GUIUtil.openWebPage("https://explorer.bisq.network/testnet/tx.html?tx=" + proposal.getTxId()));
```

Comment:
Please use 
https://github.com/bisq-network/bisq/blob/855ca062c67ca752db42cb237f0528ffec1d0b3f/desktop/src/main/java/bisq/desktop/util/GUIUtil.java#L995-L998
to open the explorer, so it also works when the DAO is live.

Reply:
Thanks, fixed it.